modules/update-upgrading-cli.adoc

```diff
@@ -37,6 +37,13 @@ NAME      VERSION   AVAILABLE   PROGRESSING   SINCE   STATUS
 version   4.9.23     True        False         158m    Cluster version is 4.9.23
 ----
 
+. Set your channel to `stable-4.10`:
++
+[source,terminal]
+----
+$ oc adm upgrade channel stable-4.10
```

Comment:
We should mention that the command might give a warning because the cluster does not have all the channel information when we run this command first time.

Reply:
@LalatenduMohanty Can we show this in example output, rather than explicitly mentioning it in a procedure?
From an end-user perspective, users will be more interested to read and execute commands. Also this information is displayed in the output , which makes them aware of the warning.

Reply:
The command does not always return warnings. So adding the warning output will  not be the right thing IMO.